google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/it/BigtableCmekIT.java

```diff
@@ -83,7 +83,7 @@ public static void validatePlatform() throws IOException {
     instanceId = AbstractTestEnv.TEST_INSTANCE_PREFIX + Instant.now().getEpochSecond();
     clusterId1 = instanceId + "-c1";
     clusterId2 = instanceId + "-c2";
-    String zoneId = testEnvRule.env().getPrimaryZone();
+    String zoneId = "us-east1-b";
```

Comment:
I dont want to have resource ids sprinkled in individual tests. AsbtractEnv is intended to hold all environment specific settings. So that if we need to make changes in your test infra, we dont have to grep for all resource identifiers in individual tests.

Easiest way I can think to do this is to expose a method in AbstractTestEnv:
```java
/** Returns test zones that can be used for intra-region testing. This might overlap with primary and secondary zones */
List<String> getMultipleZonesInSameRegion() {
   return ImmutableList.of("us-east1-b", "us-east1-c");
}
*/

And then you can use it as such:
List<String> zones = testEnvRule.env().getMultipleZonesInSameRegion();
List<String> otherZones = Sets.difference(
ImmutableSet.of(testEnvRule.env().getPrimaryRegionSecondZone(), testEnvRule.env().getSecondaryZone()

ImmutableSet.copyOf(zones))

Reply:
Okay, sounds good. I'll make the change now.